homeassistant/components/media_player/apple_tv.py

```diff
@@ -151,6 +159,22 @@ def _has_playing_media_changed(self, new_playing):
         return new_playing.media_type != old_playing.media_type or \
             new_playing.title != old_playing.title
 
+    def playstatus_error(self, updater, exception):
+        """Inform about an error and restart push updates."""
+        if isinstance(exception, _base.TimeoutError):
```

Comment:
After some thinking I believe I can remove this altogether. It was some experimenting I worked with first but I don't think it's needed.